examples/amp-video-iframe/consent.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html ⚡>
+  <head>
+    <meta charset="utf-8" />
+    <title>amp-video-iframe with amp-consent</title>
+    <link rel="canonical" href="./consent.html" />
+    <meta
+      name="viewport"
+      content="width=device-width,minimum-scale=1,initial-scale=1"
+    />
+    <script
+      async
+      custom-element="amp-consent"
+      src="https://cdn.ampproject.org/v0/amp-consent-0.1.js"
+    ></script>
+    <script
+      async
+      custom-element="amp-video-iframe"
+      src="https://cdn.ampproject.org/v0/amp-video-iframe-0.1.js"
+    ></script>
+    <style amp-custom>
+      body {
+        font-family: sans-serif;
+        max-width: 600px;
+      }
+      [fallback] {
+        font-weight: bold;
+        padding: 20px;
+        background: #fdd;
+      }
+    </style>
+    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+  </head>
+  <body>
+    <amp-video-iframe
+      id="myVideo"
+      src="/examples/amp-video-iframe/frame-consent.html"
+      width="16"
+      height="9"
+      layout="responsive"
+      autoplay
+    >
+      <amp-img
+        src="/examples/amp-video-iframe/ForBiggerJoyrides.jpg"
+        layout="fill"
+        placeholder
+      ></amp-img>
+      <amp-layout type="fill" fallback>
+        Consent rejected
+      </amp-layout>
```

Comment:
Related to https://github.com/ampproject/amphtml/issues/31568

Would love to talk more about how we can create some default or pub generated UI for elements blocked by rejected consent.